cmd/provenanced/cmd/root.go

```diff
@@ -271,24 +271,6 @@ func newApp(logger log.Logger, db dbm.DB, traceStore io.Writer, appOpts serverty
 		panic(err)
 	}
 
-	// Validate min-gas-price is a single coin.
```

Comment:
imo we don't need this check anymore since 
```
We already check that a floor gas fee is paid by every tx, hence i think the validation check for min-gas-price is irrelevant now, and an ante handler already checks that flag if some node wants to enforce higher gas price at node/mempool level already.(min_gas_prices_decorator.go)
```
and this is preventing some commands like prune from running